Makefile

```diff
@@ -59,25 +61,36 @@ $(VERSIONED_NAME): generator/generate.py generator/templates/header.py generator
 doc: $(VERSIONED_NAME)
 	-pydoctor --project-name=python-vlc --project-url=https://github.com/oaubert/python-vlc/ --make-html --verbose --html-output=doc $<
 
-test2: $(MODULE_NAME)
-	PYTHONPATH=$(VERSIONED_PATH):$(PROJECT_ROOT) python tests/test.py
-	PYTHONPATH=$(DEV_PATH):$(PROJECT_ROOT) python tests/test.py
+test_bindings2: $(MODULE_NAME)
```

Comment:
The test_* targets in Makefile depend on $(MODULE_NAME). If we try to build the module in a directory with no access to the dev includes (e.g a container where we installed libvlc-dev), $(DEV_INCLUDES) is empty, which causes an error in the $(MODULE_NAME) commands.